src/gui/tray/talkreply.cpp

```diff
@@ -0,0 +1,45 @@
+#include "talkreply.h"
+#include "accountstate.h"
+
+#include <QJsonDocument>
+#include <QJsonObject>
+#include <QJsonArray>
+
+namespace OCC {
+
+Q_LOGGING_CATEGORY(lcTalkReply, "nextcloud.gui.talkreply", QtInfoMsg)
+
+TalkReply::TalkReply(AccountState *accountState, QObject *parent)
+    : QObject(parent)
+    , _accountState(accountState)
+{
+    Q_ASSERT(_accountState && _accountState->account());
+}
+
+void TalkReply::sendReplyMessage(const QString &conversationToken, const QString &message, const QString &replyTo)
+{
+    QPointer<JsonApiJob> apiJob =  new JsonApiJob(_accountState->account(),
```

Comment:
@camilasan This is rather unusual. Why do you need both `QPointer` and a `QObject` parent for the `JsonApiJob`?